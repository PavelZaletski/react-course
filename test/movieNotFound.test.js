import renderer from 'react-test-renderer';
import Enzyme, { shallow, mount } from 'enzyme';
import React from 'react';
import MovieNotFound from '../client/js/components/movieNotFound';

describe('MovieNotFound', () => {
    it('should match snapshot', () => {
        const component = renderer.create(<MovieNotFound />).toJSON();
        expect(component).toMatchSnapshot();
    });
});