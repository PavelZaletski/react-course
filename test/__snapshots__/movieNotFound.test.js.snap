// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MovieNotFound should match snapshot 1`] = `
<div
  className="no-found"
>
  No films found
</div>
`;
